### ISSUE TITLE
Soldier zombie armor too tough?

### ISSUE THREAD

**Issue by Dunkhan** [CleverRaven/Cataclysm-DDA#22437]
Game version:0.C-24628-g8d0228

Operating system:Windows 7

Tiles or curses:Tiles

Mods active:Default

### Expected behavior
Soldier zombies, despite being harder to kill due to military gear, should not be completely immune to early-mid game cutting weapons

### Actual behavior
Firing a longbow with wooden arrows, over 20 shots, many of them crits - 0dmg
Firing a glock 19 with JHP, ~15 shots including crits - 0dmg
Stabbing with hunting knife ~5 hits - 0dmg
Firing with glock 19 with FMJ - 1-2dmg per shot, crits 3 dmg
Hitting with heavy stick - 6-7 dmg

### Steps to reproduce the behavior
Spawn all the above weapons and ammo, 
spawn some soldiers, 
use weapons on soldiers

I understand that armor balancing is a tricky issue but it is not intuitive for a stick to do more damage than a pistol, even if the soldier is wearing a kevlar vest (which most of them aren't). 
I now know that you have to carry a blunt weapon around if you are an archer build, and I probably wont die to this in future but it is a pretty disappointing way to die on a character that can mow down pretty much any other basic zombie in a 2-3 hits.

May be related to #17188

**Comment by Mecares** [comment]
Just use a bashing weapon against them and accept that some enemies require different strategies.

**Comment by Dunkhan** [comment]
If you feel that this is not a bug you should mark it accordingly and close it

**Comment by Treah** [comment]
> Firing a longbow with wooden arrows, over 20 shots, many of them crits - 0dmg
Firing a glock 19 with JHP, ~15 shots including crits - 0dmg
Stabbing with hunting knife ~5 hits - 0dmg

All of these I would expect as the zombie is wearing riot gear and these would not penetrate something like that. However the glock with FMJ should be doing a bit more damage then 3. Then again its a dead human in riot gear and the shots are just barley getting though. Also Zombies dont feel pain.  It also makes sense that the bashing weapons would work better then anything that cuts.

**Comment by Coolthulhu** [comment]
Soldier zeds are one of the few zeds whose armor actually matters. Nerfing that would make them just another generic zed, except with better drops.
Their stats are comparable to armor the player would have when wearing kevlar armor of the kind the zeds drop.

Due to how armor works (flat reduction, never percentage reduction), it's not possible to have them be significantly resistant to strong attacks without being immune to weak ones of the same type. This is actually a huge flaw with the armor system, but fixing it properly isn't trivial.

**Comment by kevingranade** [comment]
The gun performance is 100% expected with the current system.  Military standard vests are quite capable of stopping 9mm fmj with no appreciable damage to the wearer.
Likewise the arrow and knife results are expected and intended.
If anything the unintended behavior is that you can bypass so much protection by switching to using blunt force.

These are reversed if we're considering the scenario from outside the perspective of the game.  While the body armor is capable of stopping these rounds, it is not necessarily capable of doing so indefinitely.
If you e.g. unloaded a full magazine of 9mm into a zombie wearing an armored vest, the vest will degrade and start letting some rounds through.

Similarly, this armor doesn't have full coverage, you can at least in theory aim to miss the armor and bypass it entirely.  This would greatly increase the difficulty of hitting, but if the alternative is doing no damage, or having to use 30+ rounds to take down one zombie, it's an attractive option.

Likewise with a knife, as long as you know the layout of the armor, you can aim to avoid it and hit places that are unprotected.  I'd expect this to make it harder to hit, but not to the same degree as with shooting.

So in summary, there are some features we could add around armor degradation and having the player avoid hitting the armor that would make this work more like you may expect it to.

Side note, @Coolthulhu is off base here, this kind of armor should not be letting some of the damage through, from that point of view it is working exactly as intended.

**Comment by Dunkhan** [comment]
I think the fact that bashing damage gets through so easily is part of what threw me off. Military zombies are not a major danger early on as I always use the wooden sword at the start. Soldiers are basically just slightly tougher zombies with better loot. Once I progressed to significantly better weapons the fact that that suddenly resulted in death to a soldier zombie seemed incongruous. I now know that you always need a bashing weapon. Increasing their bash armor would reduce the surprise for others in future, but it might also make the soldiers too powerful.

**Comment by Mecares** [comment]
Unarmed combat deals bashing damage as far as i remember, so it is impossible to be without a bashing weapon.

**Comment by ShadowDragon8685** [comment]
This really kind of makes sense. Historically speaking, the main vulnerability of heavy armor, beyond being grounded and simply mobbed by a group, has been bashing damage. There's only so much spreading-out of kinetic energy that armor can do. To put it another way, if you hit a main in full plate mail with a sledgehammer, he's probably going down. Meanwhile, bullets of the era would likely fail to penetrate.

Seems to be working as intended. Body armor is really, really good at resisting the things that it was made to resist.

**Comment by LordZanos** [comment]
>Their stats are comparable to armor the player would have when wearing kevlar armor of the kind the zeds drop.

Zombie soldiers don't drop kevlar body armor, they have combat boots and army helmets and milspec clothing but no actual kevlar vests. Mil_armor is only dropped by Bio Operatives, I think. 

I also don't think monster armor values were adjusted when the armor formula was adjusted. A soldier zombie has 12 bash 25 cut. I think the highest armor values on anything they drop is the helmet, which is 10 bash 15 cut. And most attacks probably aren't headshots.

**Comment by Podesta** [comment]
If we wanted to be overly realistic, Kevlar is pretty crap in anything other than stopping bullets. Stabbing, etc would do an easy job out of it. Unless we assume that they improved the technology from what it does today...

But I feel like the main point is that there should have some enemies early on that the player shouldn't man up against. The soldier is especially good, because it is so easy to go around it, so it does not pose any major problems, unless the player tries to kill it very early on. For that I think the bash armor of them should be increased, until a more definitive answer could be reached, so they are not 'exploitable' from bash based damage.

**Comment by douglasdennis** [comment]
https://www.safeguardclothing.com/uk/articles/how-does-a-stab-proof-vest-work/

Second paragraph of this site refutes your statement regarding Kevlar being ineffective with stabbing attacks. The Wikipedia entry also calls out that Kevlar is commonly used for protection against cuts.

Not calling myself an expert here but I find it unlikely that a tight packing of spun fibers would be crap at dealing with a knife.

**Comment by LordZanos** [comment]
Stab resistant vests are made in a different matter than bullet resistant vests, although there are vests that are rated for both small arms fire and bladed weapons. But in any case soldier zombies don't drop any kind of kevlar vest anyway.

**Comment by Night-Pryanik** [comment]
Closing per lack of discussion for a long time and to clean up the issues list.
